### DISCUSSION TITLE
How should I write a flake.nix in order to use nix develop instead of nix develop github:informalsystems/cosmos.nix#cosmos-shell

### DISCUSSION THREAD

**Discussion by GENDRAUD** [informalsystems/cosmos.nix#112] [Q&A] [answered]
Hello 

I've writen a question in stackoverflow related to your cosmos.nix.
Wouldn't you mind trying to answer it?

https://stackoverflow.com/questions/75129830/how-should-i-write-a-flake-nix-in-order-to-use-nix-develop-instead-of-nix-develo

Best regards

**Comment by romac** [comment]
https://stackoverflow.com/a/75137097/63301

**Comment by GENDRAUD** [comment]
Your example help me to find the mistake. Thanks.
nix flake check works now. I'll write the answer after testing nix develop (tomorrow probably)

Best regards

**Comment by GENDRAUD** [comment, marked as answer]
I've tested `nix develop`. It returns an error. But exactly the same as `nix develop github:informalsystems/cosmos.nix#cosmos-shell`
I've opened an issue.

the flaxe.nix is written the stackoverflow link that I give you.
I mark this text as an answer because now `nix develop `is equivalent to `nix develop github:informalsystems/cosmos.nix#cosmos-shell`
If it doesn't work this is another problem
